test/e2e/ingress/ingress.go

```diff
@@ -863,7 +863,7 @@ spec:
 		})
 	})
 
-	Context("GatewayProxy reference Secret", func() {
+	PContext("GatewayProxy reference Secret", func() {
```

Comment:
Unlike the api7ee mode, in the apisix standalone mode, it is necessary to update the ednpoints of the gatewayproxy.